strings-csharp/StringTruncationInCSharp/StringTruncationInCSharp/StringHelperBenchmark.cs

```diff
@@ -0,0 +1,84 @@
+﻿using BenchmarkDotNet.Attributes;
+
+namespace StringTruncationInCSharp
+{
+    [MemoryDiagnoser]
+    [Orderer(BenchmarkDotNet.Order.SummaryOrderPolicy.FastestToSlowest)]
+    [RankColumn]
+    public class StringHelperBenchmark
+    {
+        private string _originalString = "This is a long string.";
+        private static readonly StringHelper _stringHelper = new StringHelper();
+
+        [Params(0.01, 0.10, 0.50, 0.95)]
+        public double MaxLengthPercentage;
+    
+        [GlobalSetup]
+        public void Setup()
+        {
+            _originalString = GetDummyStringToTestBenchmark();
+        }
+
+
+        [Benchmark]
+        public void TruncateStringUsingSubstring()
+        {
+            int maxLength = (int)(_originalString.Length * MaxLengthPercentage);
```

Comment:
We don't need this. It's possible to use int param because we have a predefined string length for the benchmark test.